Bump minimal recommended Scarb version to 2.9.4

cptartur opened the pull request:
<!-- Reference any GitHub issues resolved by this PR -->

Closes #

## Introduced changes

<!-- A brief description of the changes -->

-

## Checklist

<!-- Make sure all of these are complete -->

- [ ] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

cptartur committed:
Bump recommended Scarb version to 2.9.4

cptartur committed:
Remove logic checking scarb >= 2.9

cptartur committed:
Lint & format

cptartur committed:
scarb fmt

cptartur committed:
Update version in assert

cptartur committed:
Bump version in tests

cptartur committed:
Update outputs

cptartur committed:
Update gas related tests

ksew1 commented:
Could we run `scarb fmt` it separate PR? It is hard to spot the important changes

cptartur committed:
Revert "scarb fmt"

This reverts commit 0455c2ffd669328a07c49e3e1af385e16a63e61e.

cptartur commented:
> Could we run `scarb fmt` it separate PR? It is hard to spot the important changes

@ksew1 there you go https://github.com/foundry-rs/starknet-foundry/pull/3154

kkawula committed:
Merge branch 'master' into bump-minimal-version-2.9.4

cptartur committed:
Scarb fmt for 2.9.4 (#3154)

<!-- Reference any GitHub issues resolved by this PR -->

Closes #

## Introduced changes

<!-- A brief description of the changes -->

-

## Checklist

<!-- Make sure all of these are complete -->

- [ ] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`
